### ISSUE TITLE
Monitor batch query and streaming query I/O metrics separately

### ISSUE THREAD

**Issue by hzxa21** [risingwavelabs/risingwave#8795]

**Comment by zwang28** [comment]
IIUC, after we support dedicated serving cluster(and never serve from streaming cluster), we can achieve this separation by simply tuning grafana a bit?

**Comment by hzxa21** [comment]
> IIUC, after we support dedicated serving cluster(and never serve from streaming cluster), we can achieve this separation by simply tuning grafana a bit?

True. But I think dedicated serving cluster is optional so it may still be useful to distinguish them.

**Comment by fuyufjh** [comment]
Hi, any futher plans?

**Comment by hzxa21** [comment]
> Hi, any futher plans?

This is low priority since we can use dedicated serving cluster if we found that streaming and batch are interfering with each other. I will remove it from the milestone for now.
